io/src/main/scala/fs2/io/net/Network.scala

```diff
@@ -132,13 +132,13 @@ object Network {
           address: Option[Host],
           port: Option[Port],
           options: List[SocketOption]
-      ): Stream[F, Socket[F]] = globalSocketGroup.server(address, port, options)
+      ): Stream[F, Shared[F, Socket[F]]] = globalSocketGroup.server(address, port, options)
```

Comment:
I think both these signature can go back to the old version, and we just introduce a more general one that exposes `Shared`